coralproject/talk
# [next] Relay GraphQL Batching

## What does this PR do?

Adds GraphQL batching support that is compatible with the client side relay transport: https://github.com/relay-tools/react-relay-network-layer/blob/master/src/express-middleware/graphqlBatchHTTPWrapper.js 

## How do I test this PR?

Submit a GraphQL query:

```json
[{"query": "{ me { id } }", "id": "1"}]
```

And see the response:

```json
[{"id": "1", "payload": {"data": {"me": null}}}]
```
